p/themes/base-theme/template.css

```diff
@@ -384,6 +384,10 @@ a.btn {
 	width: 90%;
 }
 
+.alert-warn {
+	background: inherit;
+}
```

Comment:
Why do we need these lines? The `#stream:not(.alert-warn)` seems to be enough, isn't it?

Reply:
Yes indeed. It was just to make sure to have a background in all cases